.github/workflows/config.yaml

```diff
@@ -0,0 +1,37 @@
+name: Lint and Test
+
+on:
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
```

Comment:
^ I don't like this old style `requirements.txt`, can we use `Poetry` like we do for all other projects

This does not have any lock file, it's just dangerous.

Reply:
yeah, I will do that together with the `projet-template` rebase